openlibrary/plugins/recaptcha/recaptcha.py

```diff
@@ -38,6 +50,6 @@ def validate(self, value=None):
         data = r.json()
         if not data.get('success', False) and 'error-codes' in data:
             logger.error(f"Recaptcha Error: {data['error-codes']}")
-            return True
+            return accept_error(data['error-codes'])
```

Comment:
This might be more easily done with some set logic:
```suggestion
            return not (set(data['error-codes']) & INVALIDATING_ERRORS)
```

Ie check that the provided errors don't intersect the invalidating errors. But it's the same, so just a suggestion.